src/main/java/Problem1/LevelOrderTraverse.java

```diff
@@ -1,12 +1,42 @@
 package Problem1;
 
+import java.util.ArrayDeque;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Queue;
+
+/**
+ * I used an ArrayDeque because according to documentation it would be faster than a LinkedList in this
```

Comment:
I wonder what makes it faster than linkedlist?

Reply:
You inspired me to do some research on this now. I came across the below StackOverflow post which has some good discussion around this!

https://stackoverflow.com/questions/6163166/why-is-arraydeque-better-than-linkedlist#:~:text=ArrayDeque%20is%20more%20efficient%20than%20the%20LinkedList%20for,LinkedList%20implementation%20consumes%20more%20memory%20than%20the%20ArrayDeque.